pkg/cmd/pulumi/policy_new.go

```diff
@@ -337,18 +340,28 @@ func choosePolicyPackTemplate(templates []workspace.PolicyPackTemplate,
 	surveycore.DisableColor = true
 	message := "\rPlease choose a template:"
 	message = opts.Color.Colorize(colors.SpecPrompt + message + colors.Reset)
+	var selectedOption workspace.PolicyPackTemplate
+
+	for {
```

Comment:
What are we introducing this `for` loop for? I know it mirrors the last PR, but I think that `has` will always `break` out.

Reply:
Yeah I'm not actually sure why we need it in the other file as well. I can remove both.